certif/translations/en.json

```
@@ -20,11 +21,86 @@
       "gateway-timeout-error": "The service is being slow. Please try again later.",
       "internal-error": "An error occurred"
     },
-    "form": {
-      "mandatory-all-fields": "All fields are required."
+    "forms": {
+      "error-default": "The service is temporarily unavailable. Please try again later.",
+      "mandatory-fields": "The fields marked by <abbr title=\"required\" class=\"mandatory-mark\">*</abbr> are required.",
+      "mandatory-all-fields": "All fields are required.",
+      "certification-labels": {
+        "additional-certification": "Additional certification",
+        "billing-mode": {
+          "free": "Free",
+          "paid": "Payable",
+          "prepaid": "Prepaid"
+        },
+        "candidate-status": {
+          "authorized-to-resume": "Allowed to resume",
+          "finished": "Finished",
+          "ongoing": "In progress",
+          "start": "Start"
+        },
+        "email-convocation": "Email for the convocation",
+        "email-results": "Email of the results' recipient (instructor, teacher...)",
+        "external-id": "External user ID",
+        "extratime": "Extra time",
+        "extratime-percentage": "Extra time (%)",
+        "prepayment-code": "Prepayment code",
+        "pricing": "Pricing of Pix’s share"
+      },
+      "common-labels": {
+        "birth-city": "Birth city",
+        "birth-city-insee-code": "Birth city INSEE code (only for France)",
+        "birth-city-postcode": "Birth city postcode",
+        "birth-geographical-code": "Geographical birth code",
+        "birth-country": "Birth country",
+        "birth-date": "Date of birth",
+        "birth-name": "Birth name",
+        "firstname": {
+          "label": "First name",
+          "error-mandatory": "Please enter a first name."
+        },
+        "gender": {
+          "title": "Gender",
+          "man": "Man",
+          "woman": "Woman"
+        },
+        "insee-code": "INSEE code",
+        "lastname": {
+          "label": "Lastname",
```

```suggestion
          "label": "Last name",
```


Je vois aussi des “nom de naissance” (je crois avoir vu une discussion sur ce sujet dans slack d’ailleurs :o), je me demande pourquoi des fois on a l'un ou l'autre 🤔

---

Ah 😅 Pour nom de famille / nom de naissance, c'est que ce n'était pas uniformisé. On a tranché pour "Nom de naissance" (et on a 2 cas avec juste "Nom", pour le header du tableau de candidat et pour la page de login)

Je corrige pour last name